test/generation/test_utils.py

```diff
@@ -9,15 +9,40 @@
 from itertools import product
 
 import torch
+
 from botorch.acquisition import FixedFeatureAcquisitionFunction
 from botorch.generation.utils import (
+    _convert_nonlinear_inequality_constraints,
     _flip_sub_unique,
     _remove_fixed_features_from_optimization,
 )
 from botorch.utils.testing import BotorchTestCase, MockAcquisitionFunction
 
 
 class TestGenerationUtils(BotorchTestCase):
+    def test_convert_nonlinear_inequality_constraints(self):
+        for dtype in (torch.float, torch.double):
```

Comment:
Don't think you need to test for different `dtype`s here? Not used right now...

Reply:
I think it is not needed, as I am not doing anything with tensors here, so I removed it.